monero-outreach-docs/translations/pt-br/website-sections/your_privacy_pt_pt.md

```diff
@@ -0,0 +1,25 @@
+# A Sua Privacidade
+
+**“The caged bird sings with a fearful trill, of things unknown, but longed for still, and his tune is heard on the distant hill, for the caged bird sings of freedom.” - Maya Angelou - 4 de janeiro de 2021**
```

Comment:
I found the Portuguese translation for this poem in this website: https://ultimato.com.br/sites/gladircabral/2014/11/20/1118/
I think it would be nice to have it in Portuguese:
"O pássaro preso canta num gorjeio triste, sobre coisas que não conhece, mas ainda deseja, e seu canto é ouvido na colina distante, pois o pássaro preso canta sobre a liberdade."

Reply:
Nice! I'll add it!

Reply:
Done!